documentation/tut/flex-model-v2g.rst

```diff
@@ -125,6 +125,8 @@ Alternatively, to keep the car from discharging altogether during that time, lim
         }
     }
 
+.. note:: In case the soc-minima fields defines partially overlapping time periods, FlexMeasures automatically resolves this by selecting the maximum. Likewise, the minimum is selected for partially overlapping time periods in the soc-maxima, production-capacity and consumption-capacity flex-model fields, and also in the site-production-capacity and site-consumption-capacity flex-context fields.
```

Comment:
For me, the main documentation of the `soc-minima` and `soc-maxima` fields is [here](https://github.com/FlexMeasures/flexmeasures/blob/main/documentation/features/scheduling.rst?plain=1#L140), in the flex-context section of our docs.

- The note added here  might also need to be a footnote there
- Can these fields also specify ranges (like `production-capacity` above) - then we should also say that.